docs/webpack-dev-server.config.js

```diff
@@ -39,6 +39,8 @@ const config = {
   plugins: [
     // Allows for sync with browser while developing (like BrowserSync)
     new webpack.HotModuleReplacementPlugin(),
+    // Prevent moment from loading all the locales
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Comment:
No I don't think so. This isn't related to material-ui at all. It's moment. I'll add it to the README of my repo just in case anyone pays a visit.

On the other hand, anyone using persian date picker will probably have moment installed. because `moment-jalaali` - an extension - is the best tool for persian date arithmetic.